test_runner/README.md

```diff
@@ -0,0 +1,51 @@
+## Zenith test runner
+
+This directory contains integration tests.
+
+Prerequisites:
+- pytest
+    Install pytest using something like this:
+    - `pip3 install pytest` (Debian or Ubuntu)
+
+- an already compiled zenith + postgres tree
+    - See the root README.md for details
+
+
+### Running the tests
+
+At present, all the tests must be run from the git root directory.
+
+You can run individual tests like so:
+
+`pytest -s test_runner/test_pgbench.py`
+
+You can run all the tests with:
+
+`pytest -s test_runner/`
+
```

Comment:
I tried running just `pytest` at first. It was stuck in "Collecting..." stage for a long time, until I got tired and killed it. Would it be possible to have configuration file or something so that you wouldn't pass the `test_runner' dir on the command line?

Reply:
The tests are very noisy, I would not recommend `-s` by default. Although if you don't, does the output go to a log file or is it just lost?

Reply:
`pytest` by default starts searching every subdirectory of your current directory. Yes, that's a bit annoying.

My preference would be to always run `pytest` from the test_runner directory, but for some reason the `zenith` cli requires you to run it from the git root right now.

Reply:
Looks like pytest has only a few standard options for capturing stdout/stderr. Most of them seem targeted at code running inside the python interpreter.

Since we're launching each of those processes with `subprocess.run()`, we can redirect the output to a file or examine it for problems. I did a quick experiment to confirm this works, and verified it's easy to set an arbitrary file as the process stdout/stderr (it's not the python interpreter copying every byte).